.github/workflows/tests.yml

```diff
@@ -172,6 +172,12 @@ jobs:
       - name: Run tests
         run: pytest tardis ${{ env.PYTEST_FLAGS }}
         working-directory: ${{ env.TARDIS_PIP_PATH }}
+        if: always()
```

Comment:
Should this be removed before merge?

Reply:
I can, but I didn't want any other step to influence(or stop) this step. Is that the right thing to do?

Reply:
I see, so even if other installation steps fail, it will always attempt to run the tests?

Reply:
The same with the refdata generation step, it could happen that these tests fail and those pass, so I didn't want above steps to influence this, if that is alright

Reply:
yes